class MainDemoException11
{
	public static void main(String arg[])
	[
	int a=60;
	try
	{
	intc=a/arc.lenght;
	System.out.println("c:"+c);
	}
	catch(ArithmeticException e)
	{
	System.out.println("ArithmeticException");
	}
	System.out.println("After catch")
	}
}